config.tfvars

```diff
@@ -4,23 +4,37 @@
 
 # Please define the values to configure the infrastructure before install
 
+################################################################################
+# Mandatory
+################################################################################
+
 # 'environment_name' provides your environment a unique name within a single cloud provider account.
 # This value can not be altered after the configuration has been applied.
 environment_name = "<ENVIRONMENT>"
 
 # Cloud provider region that this configuration will deploy to.
 region = "<REGION>"
 
-# Custom tags for all resources to be created. Please add all tags you need to propagate among the resources.
-resource_tags = {
-  Terraform = "true"
-}
+# Bamboo license
+# To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` (i.e. `TF_VAR_bamboo_license`) and keep the below line commented out
+# If storing license as plain-text is not a concern for this environment, feel free to uncomment the following line and supply the license here
+#
+#bamboo_license = "<license key>"
```

Comment:
Can we change the license key in upper case to match with others : `<LICENSE KEY>`

Reply:
Updating this on https://github.com/atlassian-labs/data-center-terraform/pull/113